backend/dataall/modules/s3_datasets/cdk/dataset_stack.py

```diff
@@ -437,20 +460,68 @@ def __init__(self, scope, id, target_uri: str = None, **kwargs):
             },
         )
 
+        glue_sec_conf_enc_key = kms.Key(
+            self,
+            f'{glue_sec_conf_enc_key_name}',
```

Comment:
very minor --> can we just pass `glue_sec_conf_enc_key_name` instead of with f-string like `f'{glue_sec_conf_enc_key_name}',`

For both here in kms.Key using `glue_sec_conf_enc_key_name` and later on with `glue.CfnSecurityConfiguration` using `glue_sec_conf_name`

After that quick change I think this PR is good to merge!

Reply:
Done. Made both changes Noah